local-cli/generator/index.js

```diff
@@ -82,5 +82,13 @@ module.exports = yeoman.generators.NamedBase.extend({
         {name: this.name}
       );
     }
+  },
+
+  install: function() {
+    if (this.options.upgrade) {
+      return;
+    }
+
+    this.npmInstall('react', { '--save': true });
```

Comment:
Does this respect the declared version of React via `peerDependencies` (0.14.5)?